.pylintrc

```diff
@@ -2,7 +2,7 @@
 load-plugins=pylint.extensions.bad_builtin, pylint.extensions.mccabe, pylint_celery, pylint_django
 
 [MESSAGES CONTROL]
-disable=C0103, C0111, C0330, C0412, C1001, E1004, I0011, R0903, W0232, W0621
+disable=C0103, C0111, C0330, C0412, C1001, E1004, E1101, I0011, R0903, W0232, W0621
```

Comment:
What was the issue with `x has no member y`? Is it triggered by `get_permissions_enum*`?

Reply:
Enum is treated as normal class
and enum args are treated as string not as enum
which raise warning that
`str has no member value`

Reply:
done